ocm-container.sh

```diff
@@ -48,7 +48,7 @@ while [ "$1" != "" ]; do
          ;;
 
     * )
-      ARGS+=($1)
+      ARGS+=("$1")
```

Comment:
I didn't want to change it here in case I was missing something, but I found it odd that we take `ARGS` as an array here, especially when immediately below we exit out of the script with an error code if it has more than one element.  Is there a reason that `ARGS` needs to be an array?